src/pyhf/writexml.py

```diff
@@ -58,7 +60,15 @@ def indent(elem, level=0):
             elem.tail = i
 
 
-def build_measurement(measurementspec):
+def build_measurement(measurementspec, modifiertypes):
+    # need to determine prefixes
```

Comment:
As this in in the public API `pyhf.writexml.build_measurement` can we get a docstring?